manticore/platforms/evm.py

```diff
@@ -2070,7 +2121,7 @@ def _process_pending_transaction(self):
         self._pending_transaction_concretize_address()
         self._pending_transaction_concretize_caller()
         if caller not in self.accounts:
-            raise EVMException('Caller account does not exist')
+            raise EVMException("Caller account %x does not exist; valid accounts: %s" % (caller, list(map(hex, self.accounts))))
```

Comment:
We are trying to move away from %x. Do that f-magic  `f"Caller account {hex(caller)} does not exist; valid accounts: {list(map(hex, self.accounts))}" `